### PR TITLE
Delete old, obsolete formulas

### PR DESCRIPTION
**Pull request by bartekpacia**
fyi @amanjeetsingh150

### CODE REVIEW

**Comment by bartekpacia** [comment]
I suspected that maybe in the past, we suggested using `idb-companion` from our own tap, but no, we didn't: https://github.com/search?q=repo%3Amobile-dev-inc%2Fmaestro-docs%20idb-companion&type=code

File: Formula/maestro@head.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class MaestroHead < Formula
-  desc "Maestro CLI"
+  desc "The simplest and most effective mobile UI testing framework."
   homepage "https://maestro.mobile.dev"
   license "Apache-2.0"
 
   conflicts_with "mobile-dev-inc/tap/maestro", because: "This is the head version of maestro"
-  depends_on "openjdk@11"
+  depends_on "openjdk"
```
Comment (amanjeetsingh150) [inline Formula/maestro@head.rb:7]: Was this intentional?

**Review by amanjeetsingh150 — APPROVED** [review]

**Comment by amanjeetsingh150** [comment]
> I suspected that maybe in the past, we suggested using idb-companion from our own tap, but no, we didn't: https://github.com/search?q=repo%3Amobile-dev-inc%2Fmaestro-docs%20idb-companion&type=code

We don't use idb now

File: Formula/maestro@head.rb (hunk 1 above)
Comment (bartekpacia) [reply Formula/maestro@head.rb:7]: Yes, there's no need to enforce older JDK. 

It's a convention across common Formulas on homebrew - if they depend on Java, they do it like this.

**Pull request merged**